src/dialog-editor/components/box/box.html

```diff
@@ -44,15 +45,16 @@
     <div class="blank-slate-pf nosort"
          ng-if='tab.dialog_groups.length === 0'
          ng-click='vm.addBox()'>
-      <div class="blank-slate-pf-icon">
+      <div class="blank-slate-pf-icon" style="cursor: pointer;">
         <i class="fa fa-plus-circle"></i>
       </div>
-      <h1 translate>Start with adding a box</h1>
+      <h1 translate style="cursor: pointer;">Start with adding a section</h1>
     </div>
-    <div class="panel panel-default nosort">
+    <div class="add-section-box nosort">
       <a ng-click='vm.addBox()' translate>
-        <i class="pficon-add-circle-o"></i>&nbsp;Create Box
+        <i class="pficon-add-circle-o"></i>&nbsp;&nbsp;Add section
```

Comment:
Carefull not translated.
```html
<i class="pficon-add-circle-o"></i>&nbsp;&nbsp;{{__('Add section')}}
```

Reply:
Hi @karelhala, fixed this last translation issue.  Thanks for your review comments :-)